content/03.ml-concepts.md

```diff
@@ -1,24 +1,28 @@
 ## Tip 1: Concepts that apply to machine learning also apply to deep learning {#concepts}
-Deep learning is a distinct subfield of machine learning, but it is still a subfield.
-DL has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches in certain contexts, but it is not immune to the many limitations inherent to machine learning.
-Many best practices for machine learning also apply to deep learning.
-Like all computational methods, deep learning should be applied in a systematic manner that is reproducible and rigorously tested.
 
-Those developing deep learning models should select data that are relevant to the problem at hand; non-salient data can hamper performance or lead to spurious conclusions.
-Biases in testing data can also unduly influence measures of model performance, and it may be difficult to directly identify confounders from the model.
-Investigators should consider the extent to which the outcome of interest is likely to be predictable from the input data and begin by throughly inspecting the input data.
-Suppose that there are robust heritability estimates for a phenotype that suggest that the genetic contribution is modest but a deep learning model predicts the phenotype with very high accuracy.
+While deep learning has inarguably surpassed general machine learning for artificial intelligence and predictive modeling applications in terms of public awareness, deep learning still remains a subfield of machine learning.
+DL has proven to be an extremely powerful paradigm capable of outperforming "traditional" or "conventional" machine learning approaches. This is partly owed to the fact that DL algorithms are less dependent on manual feature extraction and are capable of achieving high predictive performance when operating on raw-data, such as images and text. directly.
+In contrast, traditional machine learning approaches often require manual feature extraction to produce acceptable results.
+However, the majority of best practices for machine learning still apply to deep learning.
+Like all computational methods, experiments involving deep learning should be conducted in a systematic manner and accompanied by reproducible instructions as well as rigorous model evaluation.
```

Comment:
If we are going to have this large discussion of automatic feature extraction, I think this mention of reproducible research should go before it. The mention of reproducible computational research is in line with the notion that DL is a subset of ML/computation, while the discussion of feature extraction is a more ML/DL-specific one. I think this change will help this section to flow from a general case to more ML/DL-specific scenarios, and allow the discussion of feature extraction to move more naturally into the following discussion of biases in data and evaluation.

Reply:
That's a good point! I re-organized this a bit. Actually, having the intro in mind, I think saying "DL is a subfield of machine learning" again and again may be a bit too preachery.  I think what we really just want to say is that careful model evaluation is important. Maybe see whether the reorganization makes sense to you ...

Reply:
I disagree, since "careful evaluation" is only a small subset of what we want to drive home here. If the reader remembers one thing from this tip, it should be that "DL is a subfield of ML".